File: 0884/Solution.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        String[] a1 = s1.split(" ");
        String[] a2 = s2.split(" ");
        Map<String, Integer> hm = new HashMap<>();
        for(String s : a1) hm.merge(s, 1, Integer::sum);
        for(String s : a2) hm.merge(s, 1, Integer::sum);
        List<String> res = new ArrayList<>();
        for(Map.Entry<String, Integer> e : hm.entrySet()){
            if(e.getValue() == 1) res.add(e.getKey());
        }
        return res.toArray(new String[0]);
    }
}

File: 0884/Solution.go
func uncommonFromSentences(s1 string, s2 string) (res []string) {
    list := strings.Split(s1, " ")
    list = append(list, strings.Split(s2, " ")...)
    hm := make(map[string]int)
    
    for _, v := range list {
        hm[v]++
    }

    for k, v := range hm {
        if v == 1 {
            res = append(res, k)
        }
    }
    return

}
